### DISCUSSION TITLE
List of other v2 proxies

### DISCUSSION THREAD

**Discussion by younesaassila** [younesaassila/ttv-lol-pro#300] [General]
Share other HTTP proxies that are compatible with TTV LOL PRO v2.
Want to host your own proxy? Check out the guide [here](https://wiki.cdn-perfprod.com/guides/proxy/host-your-own-proxy).

**Comment by TsukiZero** [comment]
anyone know if Luminous (EU) v2 is available for v2? I can't install the v1 extension and the v2 extension nags about v1 proxies.

**Comment by pkb8-1621521** [comment]
Hello,
I manually installed TTV LOL Pro today using these 2 proxies: 
- chrome.api.cdn-perfprod.com:4023
- chromium.api.cdn-perfprod.com:2023

But then I'm getting this error message. Any advice on how to fix this error? Thank you in advance!

![image](https://github.com/user-attachments/assets/93745ec0-7d22-4e00-8eba-2d1d775480bf)
